README.md

```diff
@@ -64,7 +61,7 @@ An [API reference](https://microsoft.github.io/MixedReality-WebRTC/api/Microsoft
 
 ## Getting Started

```

Comment:
Remove entire section. We do not want to encourage new adoptions.

Reply:
Done.